examples/README.md

```diff
@@ -0,0 +1,40 @@
+# Examples
+
+## Overview
+
+This directory contains example applications that showcase how to use RIOT-rs.
+
+- [benchmark/](./benchmark): how to use `benchmark()`
+- [embassy-net-udp/](./embassy-net-udp): UDP echo example
+- [embassy-net-tcp/](./embassy-net-tcp): TCP echo example
+- [embassy-http-server/](./embassy-http-server): HTTP server example
+- [embassy-usb-keyboard/](./embassy-usb-keyboard): USB HID example
+- [hello-world/](./hello-world): a classic
+- [minimal/](./minimal): minimized to the max RIOT-rs config
+- [threading/](./threading): how to start and use preemptively scheduled threads
+
+## Networking
+
+Some examples configure networking. By default, they will listen on a static
+IPv4 address. Be sure to configure the host computer end of the network
+accordingly.
+
+The default IPv4 address the test examples will configure is `10.42.0.61`.
+You can configure the other end like this:
+
+    $ ip address add 10.42.0.1/24 dev <DEVICE>
+    $ ip link set up dev <DEVICE>
+
+Replace `<DEVICE>` with the name of the used network device.
```

Comment:
for real beginners, maybe add here something like 

- "to find out the name of your interface you can use a command such as ifconfig". 
- "to double-check that the address has indeed be added, you can use: ip addr show dev <DEVICE>"

Reply:
Added. And replaced "DEVICE" with "INTERFACE" where appropriate.

Reply:
perfect